src/index.ts

```diff
@@ -36,4 +46,30 @@ const argv = yargs
     .alias('help', 'h')
     .argv;
 
-convert(argv.url, argv.format, argv['custom-headers']);
+if (argv['query-name'] == null) {
+    const defaultValue = "RootQueryType"
+    console.info(
+        chalk.cyan(
+            'Argument' + chalk.italic(' query-name ') +
+            'was not supplied. Using default value: ' +
+            chalk.italic.bgCyan.black(` ${defaultValue} `)
+        )
+    )
+
+    argv['query-name'] = defaultValue
+
+}
+
+if (argv['mutation-name'] == null) {
+    const defaultValue = "RootMutationType"
+    console.info(
+        chalk.cyan(
+            'Argument' + chalk.italic(' mutation-name ') +
+            'was not supplied. Using default value: ' +
+            chalk.italic.bgCyan.black(` ${defaultValue} `)
+        )
+    )
+    argv['mutation-name'] = defaultValue
+}
+
+convert(argv.url, argv.format, argv['query-name'], argv['mutation-name'], argv['custom-headers']);
```

Comment:
We can try to maintain consistency on this line in terms of how object properties are accessed, as some parts use the dot notation and the other use the 'array index' notation

Reply:
Yeah, good point. I think the easiest way to achieve consistency is moving every to 'index' notation
```suggestion
convert(argv['url'], argv['format'], argv['query-name'], argv['mutation-name'], argv['custom-headers']);
```

Reply:
Great